packages/alfa-selector/src/selector.ts

```diff
@@ -1489,6 +1491,25 @@ export namespace Selector {
     }
   }
 
+  /**
+   * {@link https://drafts.csswg.org/css-scoping-1/#selectordef-host}
+   */
+  export class Host extends Pseudo.Class<"host"> {
+    public static of(): Host {
+      return new Host();
+    }
+
+    private constructor() {
+      super("host");
+    }
+
+    public matches(_element: Element): boolean {
+      // The shadow DOM is currently not supported so there should not be any matches.
+      // If support for the shadow DOM is added, then this should be implemented.
+      return false;
+    }
```

Comment:
```suggestion
```
We can just remove it as [the super-class already does so](https://github.com/Siteimprove/alfa/blob/04373c2c37feafdea0c8926d7ac1d69538eee368/packages/alfa-selector/src/selector.ts#L751-L753) I do not feel the comment is adding that much given that it is a fairly common pattern we have for selectors we don't really support.